src/IntegratedExperts/BehatScreenshot/ScreenshotContext.php

```diff
@@ -196,6 +223,17 @@ protected function prepareDir()
         @mkdir($this->dir);
     }
 
+    /**
+     * Remove directory with previous screenshots.
+     *
+     * @param string $dir Screenshot directory name.
+     */
+    protected static function clearDir($dir)
+    {
+        clearstatcache(true, $dir);
+        @rmdir($dir);
```

Comment:
this is incorrect: we supposed to remove all screenshots in the dir, not the dir itself